server/log_rpc_server.go

```diff
@@ -434,6 +434,16 @@ func (t *TrillianLogRPCServer) GetLatestSignedLogRoot(ctx context.Context, req *
 	if err := validateGetConsistencyProofRequest(reqProof); err != nil {
 		return nil, err
 	}
+	if uint64(req.FirstTreeSize) > root.TreeSize {
+		st := status.Newf(codes.Unavailable, "first_tree_size: %v is not available, want <= %v", req.FirstTreeSize, root.TreeSize)
+		st, err := st.WithDetails(&slr)
```

Comment:
drive by: Should we rather have a proto that **contains** SLR, just in case we want to extend it with some more fields?

Reply:
It might be safer.

Reply:
Are there any other things we would like to include?